<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestAudioSetting</class>
 <widget class="QDialog" name="TestAudioSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>声音设置</string>
  </property>
  <widget class="QCheckBox" name="switchAudioRecordBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始 / 停止录制音频</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="startPlayAudioRecord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>580</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>播放 / 停播录制音频</string>
   </property>
  </widget>
  <widget class="QSlider" name="sdkAudioPlayoutVolumeSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="playoutVolumeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>设置播放音量：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="captureVolumeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置采集音量：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="sdkAudioCaptureVolumeSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="filePath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>526</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="getMusicDurationInMS_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>麦克风设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="micComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="setMusicPublishVolume_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>麦克风:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="micVolumeSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>66</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="setMusicPublishVolume_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>音量:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="micProgressBar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="micButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>麦克风测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="getMusicDurationInMS_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>扬声器设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="setMusicPublishVolume_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>音量:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="speakerProgressBar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="speakerSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>66</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="setMusicPublishVolume_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>扬声器:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="speakerButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>扬声器测试</string>
   </property>
  </widget>
  <widget class="QComboBox" name="speakerComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="getMusicDurationInMS_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SDK音量设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="getMusicDurationInMS_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>录制音频</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>361</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="capVolumeLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playVolumeLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>439</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>listView_4</zorder>
  <zorder>listView_3</zorder>
  <zorder>listView_2</zorder>
  <zorder>listView</zorder>
  <zorder>switchAudioRecordBox</zorder>
  <zorder>startPlayAudioRecord</zorder>
  <zorder>sdkAudioPlayoutVolumeSlider</zorder>
  <zorder>playoutVolumeLabel</zorder>
  <zorder>captureVolumeLabel</zorder>
  <zorder>sdkAudioCaptureVolumeSlider</zorder>
  <zorder>filePath</zorder>
  <zorder>getMusicDurationInMS_3</zorder>
  <zorder>micComboBox</zorder>
  <zorder>setMusicPublishVolume_8</zorder>
  <zorder>micVolumeSlider</zorder>
  <zorder>setMusicPublishVolume_10</zorder>
  <zorder>micProgressBar</zorder>
  <zorder>micButton</zorder>
  <zorder>getMusicDurationInMS_4</zorder>
  <zorder>setMusicPublishVolume_11</zorder>
  <zorder>speakerProgressBar</zorder>
  <zorder>speakerSlider</zorder>
  <zorder>setMusicPublishVolume_12</zorder>
  <zorder>speakerButton</zorder>
  <zorder>speakerComboBox</zorder>
  <zorder>getMusicDurationInMS_5</zorder>
  <zorder>getMusicDurationInMS_6</zorder>
  <zorder>capVolumeLabel</zorder>
  <zorder>playVolumeLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
